### ISSUE TITLE
Regarding setting the hyperparameter

### ISSUE THREAD

**Issue by arabinda-iitbbs** [lyuwenyu/RT-DETR#327]
I am using your model to finetune for my custom dataaset. can you please tell me how to mention the number of epoch, learning rate, weight decay please? Thanks in advance.

where to mention the hyperparameter in this code "torchrun --master_port=8844 --nproc_per_node=4 tools/train.py -c configs/rtdetr/rtdetr_r18vd_6x_coco.yml -t https://github.com/lyuwenyu/storage/releases/download/v0.1/rtdetr_r18vd_5x_coco_objects365_from_paddle.pth"
